contributing.md

```diff
@@ -1,9 +1,109 @@
+we put our values into practice. We expect all of our staff, contractors and contributors to know and follow this
+code.**
+
+**Our contributors and maintainers work extremely hard to build Directus as premium open-source software. Please be
+respectful of those efforts throughout our ecosystem. Trolling, harassing, insulting, or other unacceptable behavior by
+participants will not be tolerated.**
+
+In the interest of fostering an open and welcoming environment, we as pledge to make participation in our project and
+community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex
+characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality,
+personal appearance, race, religion, or sexual identity and orientation.
+
+Examples of behavior that contributes to creating a positive environment include:
+
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
+
+**Before continuing, please take a moment to read our full
+[Code of Conduct](https://github.com/directus/directus/blob/main/code_of_conduct.md).**
+
+## Ways to Contribute
+
+### Reporting Bugs
+
+If you happen to run into a bug, please post an Issue on our main GitHub Issue board:
+https://github.com/directus/directus/issues
+
+Please be as explicit and detailed as you can in the bug report. The more information is available, the easier it is for
+other contributors to help you find the solution or fix.
+
+### Leaving Feedback
+
+If you have a great idea for an improvement of the platform, or any other feedback, please make sure to open a new
+Discussion on our GitHub Discussions board: https://github.com/directus/directus/discussions
+
+### Document the Project
+
+[The Directus Docs](https://github.com/directus/docs) are a living document that can always be improved on. Any parts of
+the docs in dire need of some tender love and care? Feel free to open a Pull Request!
+
+### Helping Others
+
+The Directus community is growing quickly, which also means there's more and more people that have questions. Helping
+out your fellow developers by answering questions on [Discord](https://directus.chat) or
+[GitHub Discussions](https://github.com/directus/directus/discussions/categories/q-a) is a great way to help the
 project.
 
-Learn more about contributing to the platform: https://docs.directus.io/contributing/introduction/
+### Pull Requests
+
+#### Bug Fixes
+
+We treat Issues on the main repo as actionable items we want to get done. This also means that we welcome PRs for any
+Issue that has been labeled. Labeled issues are bugs of new features that have been triaged, accepted, and are ready to
```

Comment:
Suggest:
Labeled issues are bugs or new features

Reply:
Also, "any Issue that has been labeled." Labeled what? At all? Can we clarify this?